src/frontend/src/pages/AdminToolsPage/TeamConfig/TeamTypeTable.tsx

```diff
@@ -29,8 +29,7 @@ const TeamTypeTable: React.FC = () => {
   useEffect(() => {
     try {
       teamTypes?.forEach(async (teamType) => {
-        const imageBlob = await downloadGoogleImage(teamType.imageFileId ?? '');
-        const url = URL.createObjectURL(imageBlob);
+        const url = await getImageUrl(teamType.imageFileId ?? '');
```

Comment:
this should be in a hook that takes in the list of imageFileIds and then returns the list of imageUrls.